clerk/instance_organization_permissions.go

```diff
@@ -24,3 +25,62 @@ func (s *InstanceService) ListOrganizationPermissions(params ListInstanceOrganiz
 	}
 	return response, nil
 }
+
+type CreateInstanceOrganizationPermissionParams struct {
+	Name        string `json:"name"`
+	Key         string `json:"key"`
+	Description string `json:"description"`
+}
+
+func (s *InstanceService) CreateOrganizationPermission(params CreateInstanceOrganizationPermissionParams) (*Permission, error) {
+	req, _ := s.client.NewRequest(http.MethodPost, OrganizationPermissionsUrl)
+
+	var orgPermission Permission
+	_, err := s.client.Do(req, &orgPermission)
+	if err != nil {
+		return nil, err
+	}
+	return &orgPermission, nil
+}
+
+func (s *InstanceService) ReadOrganizationPermission(orgPermissionID string) (*Permission, error) {
+	req, err := s.client.NewRequest(http.MethodGet, fmt.Sprintf("%s/%s", OrganizationPermissionsUrl, orgPermissionID))
+	if err != nil {
+		return nil, err
+	}
+
+	var orgPermission Permission
+	_, err = s.client.Do(req, &orgPermission)
+	if err != nil {
+		return nil, err
+	}
+	return &orgPermission, nil
+}
+
+type UpdateInstanceOrganizationPermissionParams struct {
```

Comment:
:upside_down_face: Same about the name, `UpdateOrganizationPermissionParams`.

And since it's the same parameters with Create, would it make sense to have only one type, `OrganizationPermissionParams`?

Reply:
It is the same fields, except they are optional in the update params. Is there a better way to handle this given that?

Will merge this PR for now, but happy to open a new one to clean it up!